src/components/componentGroup.ts

```diff
@@ -49,12 +49,17 @@ export module Component {
       return this;
     }
 
+    protected _setSize(availableWidth: number, availableHeight: number) {
+      this._width = availableWidth;
+      this._height = availableHeight;
+    }
+
     public _isFixedWidth(): boolean {
-      return false;
+      return this.components().every((c) => c._isFixedWidth());
     }
 
     public _isFixedHeight(): boolean {
-      return false;
```

Comment:
`Group` no longer accounts for the alignments of sub`Component`s. If it did, the resultant layout behavior would be dependent on the alignments, which is contrary to the goal of having `Group` behave as though it isn't there during layout.

Since the `Group` occupies the whole cell, its own alignment has no effect.